### PR TITLE
PAYARA-1485-Create-notifier-for-New-Relic

### PR DESCRIPTION
**Pull request by mulderbaba**

### CODE REVIEW

**Comment by mulderbaba** [comment]
jenkins test please

File: appserver/payara-appserver-modules/notification-newrelic-core/src/main/java/fish/payara/notification/newrelic/NewRelicNotificationConfigurer.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                opType = RestEndpoint.OpType.POST,
+                path = "notification-newrelic-configure",
+                description = "Configures New Relic Notification Service")
+})
+public class NewRelicNotificationConfigurer extends BaseNotificationConfigurer<NewRelicNotifierConfiguration, NewRelicNotifierService> {
+
+    @Param(name = "key")
+    private String key;
+
+    @Param(name = "accountId")
```
Comment (Pandrex247) [inline appserver/payara-appserver-modules/notification-newrelic-core/src/main/java/fish/payara/notification/newrelic/NewRelicNotificationConfigurer.java]: You can use the _alias_ Param option to specify lowercase as an acceptable alternative:

`@Param(name = "accountId", alias = "accountid")`

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by smillidge** [comment]
@mulderbaba I merged the Dtadog notifier and now there are merge conflicts.

**Comment by mulderbaba** [comment]
@smillidge ok, resolved them. github now offers a cool editor for conflict resolution.

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by mulderbaba** [comment]
jenkins test please

**Pull request merged**